crates/oxc_linter/src/jest_ast_util.rs

```diff
@@ -123,6 +100,18 @@ pub fn parse_jest_fn_call<'a>(
     None
 }
 
+fn is_valid_jest_call(members: &[KnownMemberExpressionProperty], name: &str) -> bool {
+    VALID_JEST_FN_CALL_CHAINS.iter().any(|chain| {
```

Comment:
Either binary search on hand sorted data or use a hash table? The dataset is getting large.